llvm/tools/llvm-exegesis/lib/SerialSnippetGenerator.cpp

```diff
@@ -52,7 +52,7 @@ computeAliasingInstructions(const LLVMState &State, const Instruction *Instr,
       continue;
     if (OtherOpcode == Instr->Description.getOpcode())
       continue;
-    if (!State.getExegesisTarget().checkOpcodeSupported(
+    if (State.getExegesisTarget().checkOpcodeSupported(
```

Comment:
Was correct before

Reply:
Yes, now these functions are the same, I deleted `ExegesisTarget::checkOpcodeSupported`